.buildkite/scripts/lib/artifact_metadata.sh

```diff
@@ -19,8 +19,8 @@ set -euo pipefail
 #
 # $ find . -type f | sort | xargs sha256sum
 # 123veryLongSha  ./BUILD
-# 456veryLongSha  ./build_and_upload_firecracker_packages.sh
-# 789veryLongSha  ./build_and_upload_images.sh
+# 456veryLongSha  ./upload_firecracker_packages.sh
+# 789veryLongSha  ./upload_images.sh
```

Comment:
this was an unintentional replace, but since it's just an example it doesn't matter that much